### DISCUSSION TITLE
Why I can't make Disable to 2FA in my Account

### DISCUSSION THREAD

**Discussion by FatmaSorour8** [community/community#102898] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug
this button not work 
### Body
![image](https://github.com/community/community/assets/129652721/3de6a3b8-8d18-42e7-8421-c561c32a81c4)

**Comment by f-corvaro** [comment]
GitHub requires two-factor authentication (2FA) for certain actions and repositories to increase security and protect sensitive information. If you're a member of an organization that requires 2FA, or if you have access to certain sensitive repositories, you won't be able to disable 2FA. 

In your case, it seems that is required for your organization.

**Reply by FatmaSorour8** [reply]
Thanks

**Comment by DNin01** [comment, marked as answer]
If you haven't heard already, GitHub now requires 2FA for accounts that [meet certain criteria](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-mandatory-two-factor-authentication). It's an effort to protect the software supply chain.

Based on the blue banner message in your screenshot, it looks like this is the case for your account.

**Reply by FatmaSorour8** [reply]
Thanks

**Comment by Sinesio-BL** [comment]
We are in the same **2FA required** case in our GitHub organization... but we want to set up an email service account as an GitHub organization admin. Is there a way to disable / turn-off the 2FA for that kind of accounts? 

Thanks in advance
